main.py

```diff
@@ -776,16 +777,25 @@ def sizechanged(e):
 
 def makeWindowsBlur():
     from ctypes import windll
-
-    from BlurWindow.blurWindow import GlobalBlur
-
-    if theme == "Dark":
-        GlobalBlur(
-            windll.user32.GetParent(app.winfo_id()),
-            Acrylic=True,
-            Dark=True,
-            hexColor="#1c1c1c"
-        )
+    from sys import getwindowsversion
+    if  getwindowsversion().build >= 22000:
+        from win32mica import ApplyMica, MICAMODE
+        if theme == "Dark":
+            ApplyMica(HWND=windll.user32.GetParent(app.winfo_id()), ColorMode=MICAMODE.DARK)
+        else:
+            ApplyMica(HWND=windll.user32.GetParent(app.winfo_id()), ColorMode=MICAMODE.LIGHT)
+    else:
+        from BlurWindow.blurWindow import GlobalBlur
+        if theme == "Dark":
+            GlobalBlur(
+                windll.user32.GetParent(app.winfo_id()),
+                Acrylic=True,
+                Dark=True)
+        else:
+            GlobalBlur(
+                windll.user32.GetParent(app.winfo_id()),
+                Acrylic=True,
+                Dark=False,)

```

Comment:
Remove this please, we still didnt find a solution for the overexposure.

Reply:
Ok